foundry/foundry.yml

```diff
@@ -252,6 +252,16 @@ args:
         takes_value: true
         conflicts_with:
             - no-discovery
+    - snapshot-hash:
```

Comment:
Would you describe what this option does?

Reply:
Added help message to this option.